docs/topics/http/decorators.txt

```diff
@@ -71,6 +71,11 @@ a :class:`django.http.HttpResponseNotAllowed` if the conditions are not met.
 
         Support for wrapping asynchronous view functions was added.
 
+.. function:: conditional_page()
```

Comment:
I think this should be defined in the `Conditional view processing` section below rather than this section.
There is also a `Conditional View Processing` topic `docs/topics/conditional-view-processing.txt`, I'm wondering if this should be described over there too 🤔

Reply:
Hey @sarahboyce ,  I was thinking a link to `Conditional GET middleware` in  [Comparing conditional processing](https://docs.djangoproject.com/en/5.0/topics/conditional-view-processing/#comparison-with-middleware-conditional-processing) would be enough to guide someone to the `conditional page decorator`.